plugins/experimental/magick/CMakeLists.txt

```diff
@@ -16,6 +16,11 @@
 #######################
 
 add_atsplugin(magick magick.cc)
-target_link_libraries(magick PRIVATE ImageMagick::Magick++ ts::tscppapi OpenSSL::Crypto)
+
+if(DEFINED ImageMagic::MagickWand)
```

Comment:
You can use TARGET here to help readability.

Reply:
removed the if checks.